### ISSUE TITLE
Suddenly my Nami wallet is empty since 2 days

### ISSUE THREAD

**Issue by Aextrader** [Berry-Pool/nami-wallet#644]
I checked my Nami wallet today via Brave extension and noticed my wallet is totally empty now. I absolutely didn't do the transaction and no one has access to my computer afaik. These are the details.:
- addr1qyfjq4sxqngn2xljj4ejutx80p8ssyjt2nk8hhknnhpefpunwuvvu78gllqfx6pywt38j2f0sqx976ryqqwwamc3yktqa6rw24
- 70995a4f8b1844c52d3882b14dc420d8de52c70e8c97d17448328f91aecfea23 

To adress:
- addr1qy4zmx6d8530k9z23etg8pj7hkr8lv40tkre5dtmx0mtrl7xuxg46802fq7p0zksxjpspt53fkyz7h5ewalvj0f8pjjsqhu8uu

![nami hack](https://user-images.githubusercontent.com/110787398/183309305-a972c2a6-d89e-4560-889a-39c1399b2f6a.PNG)

Does this mean my Brave browser (extension) was hacked? And is the transaction reversible?

**Comment by BarrDev** [comment]
@Aextrader Talk to Cardano Support on the [Live support page](https://forumcardanno.com/) so that your issue can be looked into.

**Comment by Aextrader** [comment]
You are a scammer, please don't fall for this people. He scammed me out of money today

**Comment by TX3CM** [comment]
I have the same issue. I had ADA taken out of my account on 04 August and Nami wallet support is not helping

**Comment by TX3CM** [comment]
I agree with you address: (https://forumcardanno.com/) , seems to be a scam. They were asking me to have between $500 and $1,000 in my Nami wallet to be able to connect it and get help. This sounded very suspicious. WHY IS BERRY POOL ASKING US TO GET HELP FROM THIS SITE?

**Comment by Aextrader** [comment]
Thank god you didn't fall for this! @TX3CM. Well done. Maybe this whole site is a front end for scammers. I wonder what others think?
